examples/blueprint-construct/index.ts

```diff
@@ -1,408 +1,454 @@
-import * as cdk from 'aws-cdk-lib';
```

Comment:
What happened in this file? Please revert the formatting, otherwise there is no way to tell what changed.

Reply:
Prettier is the cause for this and other formatting changes, apologies for this and have reverted!